README.md

```diff
@@ -90,8 +90,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 The repo is also set up to generate a Docker image.
 
-To create the docker image: `docker build -t cherrytwist/client-web:0.1.1`
-
-To run a container based on the image: `docker container run -p 80:80 cherrytwist/client-web:0.1.1`
-
-And then navigate with a browser to `http://localhost:80`
+- To create the docker image: `docker build -t cherrytwist/client-web:[tag]` where [tag] can be any value
+- To add build arguments - docker build --build-arg [argument]=[value]
+  - Argument can be one of the following: ARG_GRAPHQL_ENDPOINT, ARG_AUTHENTICATION_ENABLE, ARG_AUTH_CLIENT_ID, ARG_AUTH_TENANT_ID, ARG_AUTH_API_SCOPE, ARG_AUTH_REDIRECT_URI
+- To run a container based on the image: `docker container run -p 80:80 cherrytwist/client-web:[tag]` and then navigate with a browser to `http://localhost:80`
```

Comment:
Is this for running locally only?

Reply:
Might be good to indicate that!

Reply:
@GhostOnTheFiber no, this is if you'd like to build a container with different AAD provider, or if you'd like to disable authentication in your image.